### PR TITLE
EZP-25835: Reindex location on update

### PR DESCRIPTION
**Pull request by galileo**
https://jira.ez.no/browse/EZP-25835

There were no signals fired after location update so we need to implement those to handle reindexing well.

> Status: Ready for review

### CODE REVIEW

File: eZ/Publish/API/Repository/Tests/SearchEngineIndexingTest.php [hunk 1]
```diff
@@ -168,4 +168,42 @@ public function testIndexContentWithNullField()
...
+    public function testUpdateLocation()
+    {
+        $repository = $this->getRepository();
+        $locationService = $repository->getLocationService();
+        $contentService = $repository->getContentService();
+        $searchService = $repository->getSearchService();
+
+        $rootLocationId = 2;
+        $membersContentId = 11;
+        $contentService->loadContentInfo($membersContentId);
```
Comment (pspanja) [inline eZ/Publish/API/Repository/Tests/SearchEngineIndexingTest.php]: This can be removed, together with the previous line (it is never used).

**Comment by pspanja** [comment]
Looks good to me, +1 with a linked issue and remark above fixed.

**Comment by andrerom** [comment]
and title updated ;)

**Comment by galileo** [comment]
What do you mean about the `'title updated'`?

**Comment by galileo** [comment]
What do you mean about the `'title updated'`? 
@andrerom

**Comment by galileo** [comment]
You can rereview this issue.
@pspanja @andrerom

**Comment by andrerom** [comment]
> What do you mean about the 'title updated'? 

https://github.com/ezsystems/ezpublish-kernel/pull/1653#event-665795851 :)

> You can rereview this issue.

So is it not work in progress anymore? :)

**Comment by andrerom** [comment]
+1

**Comment by andrerom** [comment]
Thanks @galileo  :)

**Pull request merged**